python/mxnet/symbol.py

```diff
@@ -1004,24 +1147,25 @@ def simple_bind(self, ctx,
 
         grad_req: string
             {'write', 'add', 'null'}, or list of str or dict of str to str, optional
-            Specifies how we should update the gradient to the args_grad.
-            - 'write' means everytime gradient is write to specified args_grad NDArray.
+            To specify how we should update the gradient to the `args_grad`.
+
+            - 'write' means everytime gradient is write to specified `args_grad` NDArray.
```

Comment:
This sentence needs revision.

"everytime" is not a word - always write "every time". 
also "is write" -> "is written"